Mid.css

```diff
@@ -0,0 +1,214 @@
+body
+{ 
+      background-color: white;
+}
+.resume
+{     
+    margin:0 auto;
+
+  width: 1024px;
+  height:1050px;
```

Comment:
no need to provide this height. try to manage this with inner content

Reply:
Done.